src/index.html

```diff
@@ -13,5 +13,159 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/ create-application" 
+    autocomplete="off"
+     method="post" onsubmit="onSubmit()">
+      <div class="blok-input">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="input-content">
+            <label class="input-label" 
+            for="Surname">
```

Comment:
```suggestion
            <label
              class="input-label" 
              for="Surname"
            >
```
Зверни увагу на форматування
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation